ai/GameDomainObjects/field.py

```diff
@@ -211,6 +214,28 @@ def __contains__(self, item: Union[Pose, Position]):
         return self.left <= item.x <= self.right and \
                self.bottom <= item.y <= self.top
 
+    # FIXME MONTREAL
```

Comment:
is that fix? @PhiBabin

Reply:
Je me souviens pas de ce que je voulais fixé. Peut-être que je trouvais le code moche

Reply:
bah a vue de nez, l'offset de 100mm sur la boundary semble un peu arbitraire, maybe valider sur notre field.

Reply:
on devrait pas bloquer le merge pour ca though, c un peu superficiel